src/crimson/os/seastore/backref/btree_backref_manager.cc

```diff
@@ -107,7 +107,7 @@ BtreeBackrefManager::get_mapping(
 	       c.trans, offset, iter.get_key(), iter.get_val());
 	return get_mapping_ret(
 	  interruptible::ready_future_marker{},
-	  iter.get_pin(c));
+	  iter.get_cursor(c));
```

Comment:
use BackrefMapping::create(), same to the other places.

Let's make it explicit where LBA/BackrefMappings are created (so we can grep the code) and do not rely on implicit conversions.